guides/source/routing.md

```diff
@@ -954,20 +1012,22 @@ root to: 'pages#main'
 root 'pages#main' # shortcut for the above
 ```
 
-You should put the `root` route at the top of the file, because it is the most popular route and should be matched first.
+You typically put the `root` route at the top of the file so that it can be matched first.
 
 NOTE: The `root` route only routes `GET` requests to the action.
```

Comment:
I'm afraid this is not true, e.g:

```ruby
  root "posts#index", via: :post
```

Generates:

```shell
Prefix Verb URI Pattern Controller#Action
  root POST /           posts#index
```

Reply:
Good catch. Updated!